Sources/CrowdinSDK/Settings/CrowdinSDK+Settings.swift

```diff
@@ -25,4 +25,9 @@ extension CrowdinSDK {
             }
         }
     }
+
+    public class func hideSettings() {
```

Comment:
I can't find where that method is called? It might be added to CrowdinSDK.stop() method so that you don't need to call it directly.

Reply:
Its just a public method for users to hide the hoverSettings but allow the SDK still be initialized.

Reply:
Another resolution could be to move the core parts to pod 'CrowdinSDK'